srv/notes-mashup.cds

```diff
@@ -14,7 +14,7 @@ entity my.bookshop.NoteableAddresses as select from API_BUSINESS_PARTNER.A_Busin
   @readonly PostalCode as postalCode,
   @readonly StreetName as street,
   @readonly HouseNumber as houseNumber,
-  notes
+  notes @(odata.contained: false)
```

Comment:
This disables containment for notes entity, right ? Why is this required ?

Reply:
Yes, because we want to have a UI that uses the list of Notes as the entry point. If containment of notes is enabled (as it is a composition of Addresses) accessing the list of all notes is no longer possible.